udaykalra/portfolio/src/main/webapp/script.js

```diff
@@ -22,15 +30,18 @@ function createCommentList(inputText) {
 /**
  * Fetches comments for display.
  */
-function getCommentData() {
-  fetch('/data')                          // sends a request to /data
-      .then(response => response.json())  // parses the response as JSON
-      .then((myComments) => {  // now we can reference the fields in myObject!
+function getCommentData(commentsLimit = 20) {
+  fetch('/data?comment-count=' + commentsLimit)  // sends a request to /data
+      .then(response => response.json())         // parses the response as JSON
+      .then((commentsPackage) => {  // now we can reference the fields as an
+                                    // object!
         const commentsElement = document.getElementById('quote-container');
         commentsElement.innerHTML = '';
-        for (var increment = 0; increment < myComments.length; increment += 1) {
-          console.log(myComments);
-          commentsElement.appendChild(createCommentList(myComments[increment]));
+        for (var increment = 0; increment < commentsPackage.length;
```

Comment:
`index` would be clearer than `increment`, since `increment` means some delta.

Reply:
Done